Sources/ETCD/proto/etcd.grpc.swift

```diff
@@ -321,6 +321,204 @@ internal enum Etcdserverpb_KVClientMetadata {
+    set { self.lock.withLockVoid { self._interceptors = newValue } }
+  }
+
+  /// Creates a client for the etcdserverpb.Watch service.
+  ///
+  /// - Parameters:
+  ///   - channel: `GRPCChannel` to the service host.
+  ///   - defaultCallOptions: Options to use for each service call if the user doesn't provide them.
+  ///   - interceptors: A factory providing interceptors for each RPC.
+  internal init(
+    channel: GRPCChannel,
+    defaultCallOptions: CallOptions = CallOptions(),
+    interceptors: Etcdserverpb_WatchClientInterceptorFactoryProtocol? = nil
+  ) {
+    self.channel = channel
+    self._defaultCallOptions = defaultCallOptions
+    self._interceptors = interceptors
+  }
+}
+
+internal struct Etcdserverpb_WatchNIOClient: Etcdserverpb_WatchClientProtocol {
+  internal var channel: GRPCChannel
+  internal var defaultCallOptions: CallOptions
+  internal var interceptors: Etcdserverpb_WatchClientInterceptorFactoryProtocol?
+
+  /// Creates a client for the etcdserverpb.Watch service.
+  ///
+  /// - Parameters:
+  ///   - channel: `GRPCChannel` to the service host.
+  ///   - defaultCallOptions: Options to use for each service call if the user doesn't provide them.
+  ///   - interceptors: A factory providing interceptors for each RPC.
+  internal init(
+    channel: GRPCChannel,
+    defaultCallOptions: CallOptions = CallOptions(),
+    interceptors: Etcdserverpb_WatchClientInterceptorFactoryProtocol? = nil
+  ) {
+    self.channel = channel
+    self.defaultCallOptions = defaultCallOptions
+    self.interceptors = interceptors
+  }
+}
+
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+internal protocol Etcdserverpb_WatchAsyncClientProtocol: GRPCClient {
+  static var serviceDescriptor: GRPCServiceDescriptor { get }
+  var interceptors: Etcdserverpb_WatchClientInterceptorFactoryProtocol? { get }
+
+  func makeWatchCall(
+    callOptions: CallOptions?
+  ) -> GRPCAsyncBidirectionalStreamingCall<Etcdserverpb_WatchRequest, Etcdserverpb_WatchResponse>
+}
+
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+extension Etcdserverpb_WatchAsyncClientProtocol {
+  internal static var serviceDescriptor: GRPCServiceDescriptor {
+    return Etcdserverpb_WatchClientMetadata.serviceDescriptor
+  }
+
+  internal var interceptors: Etcdserverpb_WatchClientInterceptorFactoryProtocol? {
+    return nil
+  }
+
+  internal func makeWatchCall(
+    callOptions: CallOptions? = nil
+  ) -> GRPCAsyncBidirectionalStreamingCall<Etcdserverpb_WatchRequest, Etcdserverpb_WatchResponse> {
+    return self.makeAsyncBidirectionalStreamingCall(
+      path: Etcdserverpb_WatchClientMetadata.Methods.watch.path,
+      callOptions: callOptions ?? self.defaultCallOptions,
+      interceptors: self.interceptors?.makeWatchInterceptors() ?? []
+    )
+  }
+}
+
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+extension Etcdserverpb_WatchAsyncClientProtocol {
+  internal func watch<RequestStream>(
```

Comment:
This is the method that we should be calling. The return type `GRPCAsyncResponseStream` is conforming to `AsyncSequence` so we should be able to wrap this.